src/window.rs

```diff
@@ -14,13 +14,34 @@ thread_local! {
     static SETTINGS: gio::Settings = gio::Settings::new(config::APP_ID);
 }
 
+#[derive(Default, Debug, Clone, Copy)]
+pub enum MarkupLanguageMode {
+    #[default]
+    Latex,
+    Typst,
+}
+
+impl MarkupLanguageMode {
+    fn get_from_settings() -> Self {
```

Comment:
```suggestion
    fn from_settings() -> Self {
```

To keep it as idiomatic Rust naming/ in-style with the rest of the codebase